apis/v1/controllers/addresses.controller.js

```diff
@@ -41,6 +53,7 @@ exports.getAddressById = async (req, res, next) => {
         const address = await Address.findById(req.params.id);
 
         if(!address) throw new Error('Address not found');
+        
 
```

Comment:
Also I am not seeing where the status property is transformed into it's string equivalent. That is, getKeyFromValue to make the address readable to the user. 
This follows through for all requests from the users.